src/Servers/Connections.Abstractions/src/IConnectionBuilder.cs

```diff
@@ -5,12 +5,27 @@
 
 namespace Microsoft.AspNetCore.Connections
 {
+    /// <summary>
+    /// Defines an interface that provides the mechanisms to configure a connection pipeline.
```

Comment:
This is some Andrew-approved copy-pasta 👍

Reply:
Seriously though, should we start all our interface summary with "Defines an interface"? Seems obvious/redundant.

Reply:
I vote @halter73 should fix up our definitions across the board.

Reply:
Do I get complete creative control? :trollface:

Reply:
You get complete creative control to react to all my wordsmithing ;)